ui/components/Icon.tsx

```diff
@@ -91,6 +97,15 @@ function getIcon(i: IconType) {
     case IconType.RemoveCircleIcon:
       return RemoveCircleIcon;
 
+    case IconType.FilterIcon:
+      return FilterIcon;
+
+    case IconType.SearchIcon:
+      return SearchIcon;
+
+    case IconType.ClearIcon:
```

Comment:
Merge conflict incoming from this PR: https://github.com/weaveworks/weave-gitops/pull/1451